open-api/immich-openapi-specs.json

```diff
@@ -7303,6 +7303,9 @@
           "isVisible": {
             "type": "boolean"
           },
+          "isWatched": {
```

Comment:
I think it makes some sense to have this be per-library, as some libraries may be static and such. That said, I agree it doesn't make a lot of sense to expose it to users. There's been some talk of making the libraries be configured and assigned from the admin panel instead, in that context it would work well.

Reply:
In the case of a static library, why are we scanning it at all? AFAIK this is an alternative approach to scanning the filesystem nightly. It could make sense to mark a library as static, so it is skipped in the future or something like that.

Reply:
It never crossed my mind to make this a global option. I just checked and Plex does it globally. There are potential arguments having it per library, but those are for potential future use cases of libraries and nothing we have today.

Reply:
Done